AUTOMATAFORMAT.md

```diff
@@ -0,0 +1,111 @@
+
+# The file format for automata
+* Based partially on Ondrej Lengal's [`.vtf' format](https://discord.com/channels/@me/864885374375821312/980792642927460372), a format of Pavol Vargovcik, and a confused disucssion.
+
+## Top-level file structure
+* The format is **line**-based. Lines can be connected by `/`.
+* Lines are parsed into tokens. Tokens are delimited by white spaces, the format is **white space sensitive**.
+* A file contains **sections** starting with a line of the form `@<SECTION-TYPE>`, containing an automaton or, in theory, anything. We will start with one section per file, containing an automaton, but it is obviously extensible to more sections and more kinds of things.
+* In automata sections, non-empty lines are **key-value lines** of the form `%<KEY> [VALUE]`, or **transition lines**. Several key-value lines with the same key mean that the key is mapped to the set of values, an occurence without a value marks the `KEY` as defined. 
+* Besides white spaces and the end of line, the following are **reserved characters**: `&`,`|`,`!`,`@`,`(`,`)`,`%`,`"`,`/`,`\` `[`,`]`,`a`,`q`,`n`,`f`,`{`,`}`.
```

Comment:
change *reserved* to  **characters with special meaning**: e.g. `a` is not reserved, it only has special meaning if present at the beginning of an identifier that has not been explicitly defined